pkg/taskrunmetrics/metrics.go

```diff
@@ -344,17 +369,33 @@ func (r *Recorder) RunningTaskRuns(ctx context.Context, lister listers.TaskRunLi
 	}
 
 	var runningTrs int
+	var trsThrottledByQuota int
+	var trsThrottledByNode int
 	for _, pr := range trs {
 		if !pr.IsDone() {
```

Comment:
can this if statement be removed?

Reply:
yep similar to the other pipelinerun discussion thread